src/front-end/src/helper-methods.js

```diff
@@ -88,6 +98,17 @@ export function marshallMenteeInfo(menteeFormValues, organization) {
   };
 }
 
+export function marshallAdminInfo(adminFormValues) {
+  return {
+    organization: adminFormValues.organizationInput,
+    email: adminFormValues.emailInput,
+    name: adminFormValues.nameInput,
+    userType: 'admin',
+  };
+}
```

Comment:
This method already existed below, but I moved it up so it was grouped with similar methods.